api/analytics/analytics_request.py

```diff
@@ -0,0 +1,58 @@
+from flask import request, jsonify
+from flask_json import JsonError
+
+from api.room.models import Room as RoomModel
+from helpers.calendar.analytics import RoomAnalytics
+from .analytic_report import AnalyticsReport
+
+
+class AnalyticsRequest():
+    """Handle analytics requests
+       :methods
+           validate_date
+           validate_request
+           get_analytic_report
+    """
+
+    def validate_date(self, timeMin, timeMax):
```

Comment:
Other parts of the app users start date and end date, would it be good to keep the consistency instead of introducing the  timeMin and timeMax